UI_Engine/Compute/Organise.cs

```diff
@@ -100,6 +122,26 @@ public static partial class Compute
             return new Output<List<SearchItem>, Tree<T>> { Item1 = list, Item2 = tree };
         }
 
+
+        /*************************************/
+        /**** Private Methods             ****/
+        /*************************************/
+
+        private static List<string> SplitPath(string path, List<string> toSkip)
+        {
+            if (path.EndsWith(")"))
+            {
+                return path.Split('.').Except(toSkip).ToList();
+            }
+            else
+            {
+                string[] split = path.Split('.');
+                return split.Concat(new string[] { split.Last() + "(...)" }).ToList();
```

Comment:
Super minor - we might use curly braces to maintain the transparency of the object initialiser syntax?
```suggestion
                return split.Concat(new string[] { split.Last() + "() {...}" }).ToList();
```

Reply:
Not sure. I feels it would be more confusing to a non-developer.